diff.go

```diff
@@ -0,0 +1,76 @@
+package main
+
+import (
+	"bytes"
+	"log"
+
+	"github.com/pmezard/go-difflib/difflib"
```

Comment:
This reminds me, we should probably start using godep on itself soon. Not part of this change, but soon.